src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageAssets.cs

```diff
@@ -1475,4 +1481,27 @@ private void ComputePackageExclusions()
             }
         }
     }
+
+    internal static class NuGetTargetFrameworkUtils
+    {
+        public static NuGetFramework GetTargetFramework(string targetFrameworkMoniker, string targetPlatformMoniker)
+        {
+            var targetFramework = NuGetUtils.ParseFrameworkName(targetFrameworkMoniker);
+            if (targetPlatformMoniker != null && !targetPlatformMoniker.Equals(string.Empty))
```

Comment:
@sfoslund will the fact that msbuild reports `,Version=` be a problem here?

Reply:
Oh, that's probably the case I was running into when I was testing that I didn't fully understand. I was getting an error with an invalid target platform when  parsing it with ParseFrameworkName. This should be fine though, because we'll just catch the error and return the target framework